GkeHub/src/V1/MembershipState.php

```diff
@@ -1,64 +1,308 @@
+        return $this->operator_state;
+    }
+
+    public function hasOperatorState()
+    {
+        return isset($this->operator_state);
+    }
+
+    public function clearOperatorState()
+    {
+        unset($this->operator_state);
+    }
+
+    /**
+     * Current install status of ACM's Operator
+     *
+     * Generated from protobuf field <code>.google.cloud.gkehub.configmanagement.v1.OperatorState operator_state = 3;</code>
+     * @param \Google\Cloud\GkeHub\ConfigManagement\V1\OperatorState $var
+     * @return $this
+     */
+    public function setOperatorState($var)
+    {
+        GPBUtil::checkMessage($var, \Google\Cloud\GkeHub\ConfigManagement\V1\OperatorState::class);
+        $this->operator_state = $var;
+
+        return $this;
+    }
+
+    /**
+     * Current sync status
+     *
+     * Generated from protobuf field <code>.google.cloud.gkehub.configmanagement.v1.ConfigSyncState config_sync_state = 4;</code>
+     * @return \Google\Cloud\GkeHub\ConfigManagement\V1\ConfigSyncState|null
+     */
+    public function getConfigSyncState()
+    {
+        return $this->config_sync_state;
+    }
+
+    public function hasConfigSyncState()
+    {
+        return isset($this->config_sync_state);
+    }
+
+    public function clearConfigSyncState()
+    {
+        unset($this->config_sync_state);
+    }
+
+    /**
+     * Current sync status
+     *
+     * Generated from protobuf field <code>.google.cloud.gkehub.configmanagement.v1.ConfigSyncState config_sync_state = 4;</code>
+     * @param \Google\Cloud\GkeHub\ConfigManagement\V1\ConfigSyncState $var
+     * @return $this
+     */
+    public function setConfigSyncState($var)
+    {
+        GPBUtil::checkMessage($var, \Google\Cloud\GkeHub\ConfigManagement\V1\ConfigSyncState::class);
+        $this->config_sync_state = $var;
+
+        return $this;
+    }
+
+    /**
+     * PolicyController status
+     *
+     * Generated from protobuf field <code>.google.cloud.gkehub.configmanagement.v1.PolicyControllerState policy_controller_state = 5;</code>
+     * @return \Google\Cloud\GkeHub\ConfigManagement\V1\PolicyControllerState|null
+     */
+    public function getPolicyControllerState()
+    {
+        return $this->policy_controller_state;
+    }
+
+    public function hasPolicyControllerState()
+    {
+        return isset($this->policy_controller_state);
+    }
+
+    public function clearPolicyControllerState()
+    {
+        unset($this->policy_controller_state);
+    }
+
+    /**
+     * PolicyController status
      *
-     * Generated from protobuf field <code>.google.cloud.gkehub.v1.MembershipState.Code code = 1 [(.google.api.field_behavior) = OUTPUT_ONLY];</code>
-     * @return int
+     * Generated from protobuf field <code>.google.cloud.gkehub.configmanagement.v1.PolicyControllerState policy_controller_state = 5;</code>
+     * @param \Google\Cloud\GkeHub\ConfigManagement\V1\PolicyControllerState $var
+     * @return $this
      */
-    public function getCode()
```

Comment:
Removed the output only attribute `code` and related public functions. Seems like a breaking change to me!

Reply:
This is a bad copy by Owlbot. We are investigating in #5247